instruments.go

```diff
@@ -35,6 +34,9 @@ const (
 	clientKey          = "client"
 	requestKey         = "request"
 	responseKey        = "response"
+	dimensionless      = "1"
+	bytes              = "By"
+	milliseconds       = "ms"
```

Comment:
Post-hoc suggestion: these are pretty common names for this sort of code, which makes it likely that we'll mistakenly shadow them in a future change. Could we change to `unitDimensionless`, `unitBytes`, and `unitMilliseconds` instead?